cardano-config/src/Cardano/Config/Shelley/Address.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Cardano.Config.Shelley.Address
```

Comment:
For addresses, as far as possible we should be trying to reuse things from our client API, since that handles addresses and (will) cover both Byron and Shelley addresses.

Showing we can reuse that module shows some sense of completeness & usefulness of that API. It's a good sanity check on that API to reuse it here.